test/xbgpu/test_engine.py

```diff
@@ -666,6 +656,8 @@ async def _send_data(
         beam_results
             Beamformer output, with shape (n_beams, n_frames,
             n_channels_per_substream, n_spectra_per_heap, COMPLEX).
+        accumulation_indices
```

Comment:
```suggestion
        acc_indices
```
for consistency with the code.

Reply:
How strange, I committed an updated `test_engine.py` in the previous commit, but this specific change didn't make it through (?). Committing now.